Sources/HeliumLogger/HeliumLogHandler.swift

```diff
@@ -0,0 +1,134 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Logging
+
+extension HeliumLogger {
+    
+    /// A one-time configuration function to bootstrap the SwiftLog logging system
+    /// using `HeliumLogger` as the logging backend.
+    ///
+    ///### Usage Example: ###
+    /// This bootstraps the SwiftLog logging system with a default
+    /// `HeliumLogger` instance. The default `HeliumLogger` instance will
+    /// be used as the logging backend for SwiftLog.
+    ///```swift
+    ///HeliumLogger.bootstrapSwiftLog()
+    ///```
+    ///
+    /// It's also possible to customize the default `HeliumLogger` instance using a configuration closure.
+    ///```swift
+    ///HeliumLogger.bootstrapSwiftLog { heliumLogger in
+    ///    heliumLogger.colored = true
+    ///}
+    ///```
+    /// - Parameter configure: An optional closure that may be used to configure the default `HeliumLogger` instance.
+    public static func bootstrapSwiftLog(_ configure: ((HeliumLogger) -> Void)? = nil) {
+        configure?(defaultLogger)
+        LoggingSystem.bootstrap(HeliumLogger.makeLogHandler)
+    }
+    
+    /// Creates a `HeliumLogHandler` instance for use with the SwiftLog logging system.
+    ///
+    ///### Usage Example: ###
+    /// This may be used to bootstrap the SwiftLog logging system with a
+    /// `HeliumLogger` instance. The `HeliumLogger` instance will
+    /// be used as the logging backend for SwiftLog.
+    ///```swift
+    ///let heliumLogger = HeliumLogger()
+    ///LoggingSystem.bootstrap(heliumLogger.makeLogHandler)
+    ///```
+    /// - Parameter label: The label to use for a SwiftLog `Logger`.
+    public func makeLogHandler(label: String) -> HeliumLogHandler {
+        return HeliumLogHandler(label: label, logger: self)
+    }
+    
+    private static let defaultLogger = HeliumLogger()
+    
+    /// Creates a `HeliumLogHandler` instance for use with the SwiftLog logging system.
+    ///
+    ///### Usage Example: ###
+    /// This may be used to bootstrap the SwiftLog logging system with a default
+    /// `HeliumLogger` instance. The default `HeliumLogger` instance will
+    /// be used as the logging backend for SwiftLog.
+    ///```swift
+    ///LoggingSystem.bootstrap(HeliumLogger.makeLogHandler)
+    ///```
+    /// - Parameter label: The label to use for a SwiftLog `Logger`.
+    public static func makeLogHandler(label: String) -> HeliumLogHandler {
+        return HeliumLogHandler(label: label, logger: defaultLogger)
+    }
+}
+
+/// A lightweight implementation of SwiftLog's `LogHandler` protocol.
+public struct HeliumLogHandler: LogHandler {
+    public var logger: HeliumLogger
```

Comment:
Is `public var` OK here? Why should the user be allowed to change it?

Reply:
Good catch. I'll change the `logger` and `label` properties to private. I don't have a reason to expose these publicly at the moment.